pyrevolve/angle/robogen/body_parts/__init__.py

```diff
@@ -1,17 +1,17 @@
 from __future__ import absolute_import
 
-from .active_hinge import ActiveHinge
-from .core_component import CoreComponent
-from .hinge import Hinge
-from .light_sensor import LightSensor
-from .touch_sensor import TouchSensor
-from .fixed_brick import FixedBrick
-from .parametric_bar_joint import ParametricBarJoint
-from .wheel import *
-from .cardan import *
-from .active_cardan import *
-from .active_rotator import *
-from .active_wheel import *
-from .active_wheg import *
+#from .active_hinge import ActiveHinge
```

Comment:
remove?

Reply:
I think we can actually remove lots from this section, given that now it lives in the direct tree encoding representation